src/extensions/mvs/load-helpers.ts

```diff
@@ -285,19 +285,21 @@ export function componentFromXProps(node: MolstarNode<'component_from_uri' | 'co
 /** Create props for `StructureRepresentation3D` transformer from a representation node. */
 export function representationProps(node: MolstarSubtree<'representation'>): Partial<StateTransformer.Params<StructureRepresentation3D>> {
     const alpha = alphaForNode(node);
-    switch (node.params.type) {
+    const params = node.params;
+    switch (params.type) {
         case 'cartoon':
             return {
-                type: { name: 'cartoon', params: { alpha } },
+                type: { name: 'cartoon', params: { alpha, tubularHelices: params.tubular_helices } },
+                sizeTheme: { name: 'uniform', params: { value: params.size_factor ?? 1 } },
```

Comment:
There is a mechanism responsible for adding default values for optional fields. So `?? 1` is superfluous here. (the same for other repr types below).